tests/assets/TestProjects/DependencyWithReadonlyClass/vendor/test/file-dependency/src/DependencyClass.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FileDependency;
+
+class DependencyClass {}
```

Comment:
Shouldn't this be readonly?

```suggestion
readonly class DependencyClass {}
```

Reply:
I tried but it seams like this will break on PHP < 8.2 as this is somehow interpreted by PHP runtime and not php-parser :/

Reply:
So it's solved now? :)

Reply:
Yes :)